src/WebJobs.Script.WebHost/Security/Authentication/Jwt/ScriptJwtBearerExtensions.cs

```diff
@@ -31,8 +30,16 @@ public static AuthenticationBuilder AddScriptJwtBearer(this AuthenticationBuilde
                             {
                                 OnMessageReceived = c =>
                                 {
+                                    // By default, tokens are passed via the standard Authorization Bearer header. However we also support
+                                    // passing tokens via the x-ms-site-token header.
+                                    if (c.Request.Headers.TryGetValue(ScriptConstants.SiteTokenHeaderName, out StringValues values))
+                                    {
+                                        // the token we set here will be the one used - Authorization header won't be checked.
+                                        c.Token = values.FirstOrDefault();
```

Comment:
For my own understanding: do we expect this token to be the JWT directly, or will it be like `Authorization` header where it is `Authorization: Bearer {token}`?